### ISSUE TITLE
status development

### ISSUE THREAD

**Issue by llronaldoll** [Muqsit/mcMMO#4]
this plugin is still under development, or was it abandoned? please add       "miner" 

![mcmmo](https://user-images.githubusercontent.com/5334591/43694576-5fecdd16-990a-11e8-813c-59b3aa0f187c.png)

**Comment by Muqsit** [comment]
I thought of working on it later but got cut away due to other projects, I may work on it sometime. Anyway, feel free to copy it, I don't really mind the license.

**Comment by llronaldoll** [comment]
yes, I really wanted to follow this project, but I do not understand anything about programming

**Comment by llronaldoll** [comment]
I hope one day you will have some time to devote to this plugin that was getting incredible

**Comment by llronaldoll** [comment]
could only help me with the question of the plugin not saving the progress of the player?

**Comment by jasin** [comment]
I picked this up and started working on it to learn plugin development for PM. I created a new repository instead of forking because of the way I develop.

Credit was given to original author. It has been updated to work with API 3.0.0 

Let me know if there is anything I need to do to comply [Repo](https://github.com/jasin/mcMMO)

**Comment by Muqsit** [comment]
@jasin can you email me `notmuqsit@gmail.com`
